tests/test_conversation_manager.py

```
@@ -0,0 +1,652 @@
+    )
+
+
+@pytest.fixture
+def temp_files():
+    """Fixture providing temporary test files"""
+    with tempfile.TemporaryDirectory() as temp_dir:
+        # Create test files
+        test_files = {}
+
+        main_file = os.path.join(temp_dir, "main.py")
+        with open(main_file, "w") as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)